.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Build and Verify
+# Triggers the workflow on push or pull request events
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: "17"
+
+      - name: Install dependencies
+        run: npm i
+
+      - name: Install rc-apps cli
+        run: npm i -g @rocket.chat/apps-cli
```

Comment:
@csmith14 We should our fork of `apps-cli` here https://github.com/WideChat/Rocket.Chat.Apps-cli

Reply:
The [WideChat/Rocket.Chat.Apps-cli](https://github.com/WideChat/Rocket.Chat.Apps-cli) package fails to install, with a `TAR_ENTRY_ERROR ENOENT`:

https://github.com/WideChat/Apps.Salesforce.LiveAgent/runs/6512327350?check_suite_focus=true

![image](https://user-images.githubusercontent.com/27293306/170066816-4c702316-fd28-440f-80ee-4e5103506ac6.png)